xrpl/transaction/amm_delete.go

```diff
@@ -0,0 +1,69 @@
+package transaction
+
+import (
+	"github.com/Peersyst/xrpl-go/xrpl/ledger-entry-types"
+)
+
+// Delete an empty Automated Market Maker (AMM) instance that could not be fully deleted automatically.
+// Normally, an AMMWithdraw transaction automatically deletes an AMM and all associated ledger entries when it withdraws all the assets from the AMM's pool.
+// However, if there are too many trust lines to the AMM account to remove in one transaction, it may stop before fully removing the AMM.
+// Similarly, an AMMDelete transaction removes up to a maximum of 512 trust lines; it may take several AMMDelete transactions to delete all the trust lines and the associated AMM.
+// In all cases, only the last such transaction deletes the AMM and AccountRoot ledger entries.
+//
```

Comment:
To display the example as a json, wrap the comment with ```json {content} ``` and it will be displayed as a json 👍🏻

Reply:
Thanks, I added it.